webapp/src/main/java/com/box/l10n/mojito/pagerduty/PagerDutyClient.java

```diff
@@ -0,0 +1,113 @@
+package com.box.l10n.mojito.pagerduty;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.text.MessageFormat;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * PagerDuty client for creating (triggering) and resolving incidents.
+ *
+ * @author mattwilshire
+ */
+public class PagerDutyClient {
+
+  static Logger logger = LoggerFactory.getLogger(PagerDutyClient.class);
+
+  static final String BASE_URL = "https://events.pagerduty.com";
+  static final String ENQUEUE_PATH = "/v2/enqueue";
+
+  public static final int MAX_RETRIES = 3;
```

Comment:
Should we make this configurable, with a default value of 3?

Reply:
Done, I added a PagerDutyRetryConfiguration class that has default values and if there are values specified in the application.properties it will use those.